Error: Cannot find module 'rx'

```
mac:~ luckydonald$ pipboycli
module.js:338
    throw err;
    ^

Error: Cannot find module 'rx'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/pipboycli/lib/index.js:10:11)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:475:10)
```

Definitely reproducible on 4.x, ran this in a fresh docker container. :sob:

If I upgrade `npm`, no problem:

```
$ docker run --net=host --rm -it node:4.1 bash
root@default:/# npm --version
2.14.4
root@default:/# npm install -g pipboycli > /dev/null 2>&1
root@default:/# pipboycli
module.js:338
    throw err;
    ^

Error: Cannot find module 'rx'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/pipboycli/lib/index.js:10:11)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:475:10)
root@default:/# npm install -g npm > /dev/null 2>&1
root@default:/# npm install -g pipboycli > /dev/null 2>&1
root@default:/# pipboycli
... yay display
```

--- a/package.json
+++ b/package.json
@@ -26,7 +26,8 @@
   "homepage": "https://github.com/RobCoIndustries/pipboycli#readme",
   "dependencies": {
     "blessed": "^0.1.81",
-    "pipboylib": "^1.1.0"
+    "pipboylib": "^1.1.0",
+    "rx": "^4.0.7"
   },
   "devDependencies": {
     "babel": "^6.1.18",

